### PR TITLE
Add a notice for table attributes on 5.4.0

### PR DESCRIPTION
**Pull request by en-jin19**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
Add a notice for table attributes on 5.4.0

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v6.0 (TiDB 6.0 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/9017
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: table-attributes.md [hunk 1]
```diff
@@ -11,6 +11,8 @@ The Table Attributes feature is introduced in TiDB v5.3.0. Using this feature, y
 >
 > - Currently, TiDB only supports adding the `merge_option` attribute to a table or partition to control the Region merge behavior. The `merge_option` attribute is only part of how to deal with hotspots. For more information, refer to [Troubleshoot Hotspot Issues](/troubleshoot-hot-spot-issues.md).
 > - When you use TiDB Binlog or TiCDC to perform replication or use BR to perform incremental backup, the replication or backup operations skip the DDL statement that sets table attributes. To use table attributes in the downstream or in the backup cluster, you need to manually execute the DDL statement in the downstream or in the backup cluster.
+> - TiDB v5.4.0 **does not support** Index Region. For details of Index Region, refer to [Split Index Region](/sql-statements/sql-statement-split-region.md#split-index-region).
+> - In TiDB v5.4.0, adding table attributes **only** takes effect for existing partitions. To add or delete partitions after adding table attributes, you need to reset this feature. For detailed methods, see [How to use table attributes](#usage).
```
Comment (TomShawn) [inline table-attributes.md:15]:
```suggestion
> - In TiDB v5.4.0, adding table attributes **only** takes effect on existing partitions. To add or delete partitions after adding table attributes, you need to reset table attributes. For detailed methods, see [How to use table attributes](#usage).
```

**Review by TomShawn — APPROVED** [review]
rest LGTM

**Review by rleungx — APPROVED** [review]

**Pull request merged**